ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml

```diff
@@ -4384,6 +4384,76 @@ tests:
   steps:
     cluster_profile: nutanix-qe-dis
     workflow: agent-qe-nutanix-disconnected
+- as: aws-short-cert-rotation-f9
+  cron: 42 11 2,11,23 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+    test:
+    - chain: openshift-e2e-test-qe-cert-rotation
+    workflow: openshift-e2e-cert-rotation-short
+- as: gcp-short-cert-rotation-f9
+  cron: 4 22 5,14,26 * *
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      FEATURE_GATES: '["ShortCertRotation=true"]'
+      FEATURE_SET: CustomNoUpgrade
+    test:
+    - chain: openshift-e2e-test-qe-cert-rotation
+    workflow: cucushift-installer-rehearse-gcp-ipi
+- as: azure-short-cert-rotation-f9
+  cron: 46 4 9,18,30 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      FEATURE_SET: TechPreviewNoUpgrade
```

Comment:
For the failure about 'could not determine environment flags: environment variable AZURE_AUTH_LOCATION is not set' , could you add the env  AZURE_AUTH_LOCATION here ?